sql/core/src/test/scala/org/apache/spark/sql/SparkSessionExtensionSuite.scala

```diff
@@ -88,6 +88,12 @@ class SparkSessionExtensionSuite extends SparkFunSuite {
     }
   }
 
+  test("inject data source rewrite rule") {
```

Comment:
Shall we add a JIRA prefix? That's guided in http://spark.apache.org/contributing.html FWIW.

Reply:
Sure, I can do that. If I remember correctly, Spark used to prepend JIRA prefixes in test names only for bug fixes. I think I was asked to remove such prefixes one or two times before. I believe there was a recent discussion on the dev list regarding this point but I don't recall the summary. Could you fill me in on the current strategy, @HyukjinKwon? The guide says there should be JIRA prefix every time we add more tests to existing suites. Do I understand that correctly?

Reply:
It's:

> Also, you should consider writing a JIRA ID in the tests when your pull request targets to fix a specific issue. In practice, usually it is added when a JIRA type is a bug or a PR adds a couple of tests to an existing test class. 

But I am rather just suggesting to add the prefix rather than saying this is a rule to follow :-).

Reply:
Done.